README.md

```diff
@@ -34,46 +35,103 @@ snowdrop	    https://snowdrop.github.io/helm
 
 ## Available Charts in Repository
```

Comment:
`Available Charts in Repository` ==> `List of charts available`

Reply:
Changed